src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibratorDataflow2IntegrationTest.java

```diff
@@ -104,68 +103,58 @@ public void runAllTests() throws Exception {
 
     @DataProvider(name = "BQSRTest")
     public Object[][] createBQSRTestData() {
-        final String hg18Reference = publicTestDir + "human_g1k_v37.chr17_1Mb.fasta";
-        final String b36Reference = getResourceDir() + "human_b36_both.chr1_1k.fasta";
-        final String HiSeqBam = getResourceDir() + "NA12878.chr17_69k_70k.dictFix.bam";
+        // we need an API key to get to the reference
+        final String apiArgs = "--apiKey " + getDataflowTestApiKey() + " --project " + getDataflowTestProject() + " ";
+        final String localResources =  getResourceDir();
+        final String hg19Ref = "EMWV_ZfLxrDY-wE";
+        //final String GRCh37RefLite = "EJjur6DxjIa6KQ"; // that's the "lite" version
+        final String GRCh37Ref = "EOSsjdnTicvzwAE"; // that's the "full" version
```

Comment:
Could you declaure these reference ids as public constants in `RefAPISource` so that we can use them elsewhere?